tools/benchmark/src/test/Benchmark.unit.tests.ts

```diff
@@ -125,6 +130,79 @@ describe("`benchmark` function", () => {
 			type: BenchmarkType.OwnCorrectness,
 		});
 	}
+
+	// This pattern is roughly what the non-custom `benchmark` does.
+	// It minimizes per iteration over head by timing the whole batch as a single unit.
+	// This is important for timing operations which are very fast relative to measurement overhead and clock precision.
+	// Since measurement overhead and clock precision are not the same on all systems, this approach necessary to be robustly portable.
```

Comment:
```suggestion
	// Since measurement overhead and clock precision are not the same on all systems, this approach needs to be robustly portable.
```

Reply:
```suggestion
	// Since measurement overhead and clock precision are not the same on all systems, this approach is necessary to be robustly portable.
```